test/parallel/forall/const-indices.chpl

```diff
@@ -0,0 +1,16 @@
+var A, B: [1..10] real;
+
+forall i in {1..10} {
+  ref a = A[i];
+  i = i+1;
+  a = i;
+}
+
+forall i in {1..10} {
+  ref b = B[i];
```

Comment:
Is one of these loops supposed to loop over a range?

Reply:
Fixed in PR #13099.